Framework/CurveFitting/src/Algorithms/QENSFitSequential.cpp

```diff
@@ -278,6 +279,36 @@ createGroup(const std::vector<MatrixWorkspace_sptr> &workspaces) {
     group->addWorkspace(workspace);
   return group;
 }
+
+IAlgorithm_sptr createProcessIndirectFitParametersAlgorithm(
+    ITableWorkspace_sptr parameterWorkspace, const std::string &xAxisUnit,
+    const std::vector<std::string> &parameterNames) {
+  auto pifp =
+      AlgorithmManager::Instance().create("ProcessIndirectFitParameters");
+  pifp->setAlwaysStoreInADS(false);
+  pifp->setProperty("InputWorkspace", parameterWorkspace);
+  pifp->setProperty("ColumnX", "axis-1");
+  pifp->setProperty("XAxisUnit", xAxisUnit);
+  pifp->setProperty("ParameterNames", parameterNames);
+  return pifp;
```

Comment:
As is this isn't created as a child, it will be included in the History as a parent history (causing issues for project recovery).